SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLoadBalancingHandler.cpp

```diff
@@ -292,3 +293,46 @@ VirtualWorkerId FSpatialLoadBalancingHandler::GetWorkerId(const AActor* NetOwner
 	}
 	return NewAuthVirtualWorkerId;
 }
+
+void FSpatialLoadBalancingHandler::UpdateActorsHandedOver(USpatialNetDriver& InNetDriver,
+														  const TSet<Worker_EntityId_Key>& EntitiesHandedOver,
+														  const TSet<AActor*>& ActorsHandedOver)
+{
+	const bool bStrategyWorkerEnabled = USpatialStatics::IsStrategyWorkerEnabled();
+	const bool bDirectAssignment = bStrategyWorkerEnabled && !InNetDriver.LoadBalanceStrategy->IsStrategyWorkerAware();
+
+	if (!ensure(bStrategyWorkerEnabled))
+	{
+		return;
+	}
+
+	if (!bDirectAssignment)
+	{
+		for (AActor* Actor : ActorsHandedOver)
+		{
+			// If we're setting a different authority intent, preemptively changed to ROLE_SimulatedProxy
+			Actor->Role = ROLE_SimulatedProxy;
+			Actor->RemoteRole = ROLE_Authority;
+
+			Actor->OnAuthorityLost();
+		}
+
+		for (auto EntityId : InNetDriver.HandoverManager->GetActorsToCheckForAuth())
```

Comment:
```suggestion
		for (Worker_EntityId_Key EntityId : InNetDriver.HandoverManager->GetActorsToCheckForAuth())
```
minor: no need for auto here?